consensus/validation.go

```diff
@@ -725,6 +725,8 @@ func validateV2FileContracts(ms *MidState, txn types.V2Transaction) error {
 
 	validateContract := func(fc types.V2FileContract, renewal bool) error {
 		switch {
+		case fc.Filesize > fc.Capacity:
+			return fmt.Errorf("has filesize (%v) exceeding capacity (%v)", fc.Filesize, fc.Capacity)
```

Comment:
we could make this stricter -- `fc.Filesize != fc.Capacity`. There might be a reason to create a contract with extra capacity, though, idk.

Reply:
I can't think of a good reason to restrict it and there might be a usecase for allowing it.